src/univalue/gen/gen.cpp

```diff
@@ -1,84 +0,0 @@
-// Copyright 2014 BitPay Inc.
-// Distributed under the MIT software license, see the accompanying
-// file COPYING or https://opensource.org/licenses/mit-license.php.
-
-//
-// To re-create univalue_escapes.h:
-// $ g++ -o gen gen.cpp
-// $ ./gen > univalue_escapes.h
```

Comment:
I wonder if this file should stay? I know it is unused by the build system, but it may be used by devs (in the unlikely case that the header needs to be re-generated)?

Reply:
If someone wants to regenerate the header (it hasn't changed in the last 7 years besides the `NULL` -> `nullptr` refactor), I think they could retreive the file from git. I'd rather not keep un-(likely-to-be)-used/tested files around. If the header needs to be changed, it could also just be modified directly.